xmtp_mls/src/subscriptions.rs

```diff
@@ -143,6 +164,20 @@ where
                     None
                 }
             }
+        };
```

Comment:
Possibly duplicate code and should be removed. Probably part of my bad merge.

Reply:
yep, i think so. `event_queue` is defined below on 170.